src/main/java/com/ufcg/es/biblioconex/controller/LoginController.java

```diff
@@ -0,0 +1,38 @@
+package com.ufcg.es.biblioconex.controller;
+
+import com.ufcg.es.biblioconex.dto.LivroDTO;
+import com.ufcg.es.biblioconex.dto.UsuarioDTO;
+import com.ufcg.es.biblioconex.service.UsuarioService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(
+        value = "/api/login",
+        produces = MediaType.APPLICATION_JSON_VALUE
+)
+public class LoginController {
+    @Autowired
+    UsuarioService usuarioService;
+
+    @PostMapping("/cadastrar")
+    public ResponseEntity<?> cadastrarUsuario(
+            @RequestBody @Valid UsuarioDTO usuarioDTO,
+            @RequestParam String login, @RequestParam String senha) {
```

Comment:
Não funcionou aqui, não entendi porque recebe login e senha no link e no corpo da requisição, deixar apenas no corpo.
```suggestion
             {
```

Reply:
é pedido 2 vezes pq o do DTO é o que vai ser inserido na tabela e os do corpo são as de autenticação. Porque só o bibliotecario cadastra. Alterei os nomes pra "loginBibliotecário" e "senhaBibliotecario" pra ficar mais claro